docs/csharp/getting-started/with-visual-studio-code.md

```diff
@@ -13,16 +13,45 @@ ms.assetid: 76c23597-4cf9-467e-8a47-0c3703ce37e7
 
 ---
 
-# 🔧 Getting started with Visual Studio Code
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/944) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# Getting started with Visual Studio Code
+
+.NET Core gives you a fast and modular platform for creating server applications that run on Windows, Linux and macOS. Use Visual Studio Code with the C# extension to get a powerful editing experience with full support for C# IntelliSense (smart code completion) and debugging.
+
+## Getting Started
```

Comment:
The problem is this is a VERY specific scenario. .NET CLI tools are the main getting-started scenario for .NET. cross-platform. But this guide is only for .NET, C# in **VS Code**.

The cross-plat tutorial for .NET is already [here](https://docs.microsoft.com/en-us/dotnet/articles/core/tutorials/using-with-xplat-cli)

I don't mind if we add it under .NET Core Guide instead as a "Get started with Visual Studio Code" topic. If we did not include it in the C# guide would we also delete the "Getting Started with Visual Studio" ?

I think a lot of users who end up here are going to be trying to write C# on their mac or linux. It makes sense to have it under the C# Getting Started section in that scenario.